client/platform/desktop/frontend/store/dataset.ts

```diff
@@ -96,6 +98,8 @@ function setRecents(meta: JsonMeta) {
     originalBasePath: meta.originalBasePath,
     originalVideoFile: meta.originalVideoFile,
     transcodedVideoFile: meta.transcodedVideoFile,
+    multiCam: !!meta.multiCam,
```

Comment:
This meta is returned from the `loadMetadata` which return the underlying type instead of declaring it as `multi`.  I'll put a TODO here to swap it when we officially add `'multi'` to `DatasetType`.

I'll throw an error in loadMetadata so that way it will error out of it attempts to load `multi` type data with no multiCam.